src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -152,39 +175,94 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                     <Typography variant="h2" gutterBottom>
                       {t('registration.files_and_license.files')}
                     </Typography>
+
                     {files.length > 0 && (
                       <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mb: '2rem' }}>
-                        {files.map((file) => {
-                          const associatedFileIndex = associatedArtifacts.findIndex((artifact) => {
-                            if (associatedArtifactIsFile(artifact)) {
-                              const associatedFile = artifact as AssociatedFile;
-                              return associatedFile.identifier === file.identifier;
-                            }
-                            return false;
-                          });
+                        <TableContainer component={Paper}>
+                          <Table sx={alternatingTableRowColor}>
+                            <TableHead sx={{ fontWeight: 600 }}>
+                              <TableRow>
+                                <TableCell>
+                                  <Typography fontWeight="bold">{t('common.name')}</Typography>
```

Comment:
Har du sett på muligheten til å sette teksten i TableHead til bold i theme? For vi har en tabell i contributors også, men der er det bare brukt default, slik at den ser annerledes ut nå